vnext/codegen/NativeAsyncSQLiteDBStorageSpec.g.h

```diff
@@ -0,0 +1,63 @@
+
+/*
+ * This file is auto-generated from a NativeModule spec file in js.
+ *
+ * This is a C++ Spec class that should be used with MakeTurboModuleProvider to register native modules
+ * in a way that also verifies at compile time that the native module matches the interface required
+ * by the TurboModule JS spec.
+ */
+#pragma once
+
+#include "NativeModules.h"
+#include <tuple>
+
+namespace Microsoft::ReactNativeSpecs {
+
+struct AsyncSQLiteDBStorageSpec : winrt::Microsoft::ReactNative::TurboModuleSpec {
```

Comment:
>Async [](start = 7, length = 5)

I know this is from FB, but shouldn't all Turbo Modules be Async so isn't naming this superfluous?

Reply:
This spec is for a backend implementation of the `AsyncStorage` API using SQLite, which is probably why it has async in the name.

TurboModules are actually synchronous though. The JS engine executes them directly on the JS thread through JSI. This is in contrast to BatchedBridge native modules we have right now which serialize args and pass them to a different thread.